Dockerfile

```diff
@@ -36,7 +39,45 @@ RUN addgroup --system --gid $IMMU_GID immu && \
 
 EXPOSE 3322
 EXPOSE 9497
+EXPOSE 8080
 
 HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 CMD [ "/usr/local/bin/immuadmin", "status" ]
 USER immu
 ENTRYPOINT ["/usr/sbin/immudb"]
+
+
+FROM scratch as scratch
+MAINTAINER CodeNotary, Inc. <info@codenotary.io>
+
+ARG IMMU_UID="3322"
+ARG IMMU_GID="3322"
+
+ENV IMMUDB_HOME="/usr/share/immudb" \
+    IMMUDB_DIR="/var/lib/immudb" \
+    IMMUDB_DBNAME="immudb" \
```

Comment:
hmm, I think `IMMUDB_DBNAME` is not used....

Reply:
Didn't find this option in immudb. Removed